Twisted-Tutorial.ipynb

```diff
@@ -403,6 +567,13 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally we show how to directly create, and fire, deferreds. This is, in essence, a minimal chat engine. The \"/block\" is a long poll, waiting for a message to come in. /unblock/ sends a message to the client that has been listening the longest. Note that because Twisted is asynchronous, \"longest\" is well defined."
```

Comment:
The fact that longest is well defined is an awesome point to make.  I think it'd be good to explain what that definition is.  Maybe like so:

"..."longest" is well defined, being exactly the first position in a list that grows in length with each request."